backend/app/api/api_v1/routers/lookups/router.py

```diff
@@ -0,0 +1,3 @@
+from fastapi import APIRouter
+
+lookups_router = APIRouter()
```

Comment:
we could just put this in `main.py` I think

Reply:
If you don't mind - I'll leave it where it is :smile: